src/pages/TCY/components/Stake/StakeInput.tsx

```diff
@@ -0,0 +1,121 @@
+import { Button, CardFooter, HStack, Stack } from '@chakra-ui/react'
+import type { AssetId } from '@shapeshiftoss/caip'
+import { ethAssetId } from '@shapeshiftoss/caip'
```

Comment:
Probably should drill that down since it already exists up top in header, but doesn't matter much since this will be wired up after, no need to!

FYI, I'm not entirely sure if we will be able to represent TCY as an asset (probably should)

Reply:
TCY will be an asset native to Thorchain. So since it will have markets it should be represented as an asset right?

Reply:
bet
<img width="1398" alt="image" src="https://github.com/user-attachments/assets/8a6dc0cf-7ba3-47a2-8c0c-994eab82cb47" />